apps/cowswap-frontend/src/legacy/hooks/useActivityDerivedState.ts

```diff
@@ -118,7 +118,14 @@ export function getActivityLinkUrl(params: {
   const { chainId, id, enhancedTransaction, order } = params
 
   if (enhancedTransaction) {
-    const { transactionHash, safeTransaction } = enhancedTransaction
+    const { transactionHash, hash, safeTransaction, hashType } = enhancedTransaction
+
+    /**
+     * This is a special case for Gnosis Safe transactions created via WC in a Safe with 1/1 signers
+     */
+    if (hashType === HashType.GNOSIS_SAFE_TX && hash) {
+      return getEtherscanLink(chainId, 'transaction', hash)
```

Comment:
Is this right? if the hash is a safe TX, we show etherscan link?

Reply:
lol, I had the same question https://github.com/cowprotocol/cowswap/pull/4174#discussion_r1555619972

Reply:
I see, thanks